### PR TITLE
feat(playground): redesign vrl playground and make responsive

### PR DESCRIPTION
**Pull request by britton-from-notion**
# Changes

- Adds responsive media queries to the VRL playground
- Redesigns the VRL playground with aesthetic refresh

# After
Here's what the re-design looks like
image
image
image

# Before
Here's the original
image
image

### CODE REVIEW

**Comment by pront** [comment]
Hi @britton-from-notion, thank you for this PR! We will take a look shortly.

**Comment by pront** [comment]
Two small requests:
* can we keep the purple box background? 
image
* (optional, TIOLI) add a datadog logo?

**Comment by pront** [comment]
Oh another thing, if the branch name contains `website` a preview will be generated.

**Comment by britton-from-notion** [comment]
@pront TIL renaming the branch closes the PR

**Comment by pront** [comment]
> @pront TIL renaming the branch closes the PR, woops!

No problem :)

> Yeah I can totally add the purple background in. 
Thanks!

> I might skip the datadog logo one for now and do that in a follow up just cuz I don't have any clever ideas on where to put it or which image to use rn!
👍 For future reference: https://www.datadoghq.com/about/resources/

**Comment by britton-from-notion** [comment]
@pront how do you feel about purple top bar like this? I can also do the boxes but wanted to double check

image

**Comment by pront** [comment]
> @pront how do you feel about purple top bar like this? I can also do the boxes but wanted to double check
> 
> image

Thanks for updating! I slightly prefer grid like boxes (similar to how it is today) but TIOLI. The rest of the change look great. I will try to find some time build this PR locally tomorrow.

**Comment by britton-from-notion** [comment]
Here's the updated version with the grid box header items more closely in line with the original look while still being a bit lighter-weight.

image
image

File: lib/vector-vrl/web-playground/public/index.css [hunk 1]
```diff
@@ -1,329 +1,385 @@
...
+  --light-grey: #F5F5F5;
 }
 
-table {
-  width: 100%;
-  border-spacing: 10px;
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
```
Comment (pront) [inline lib/vector-vrl/web-playground/public/index.css:17]: You can add this word to `.github/actions/spelling/allow.txt`

**Review by pront — APPROVED** [review]
Thank you @britton-from-notion, this is a significant improvement!

One last extreme nitpick (feel free to skip this one):
image
can we make these margins equal?

**Comment by britton-from-notion** [comment]
Hey again @pront! Gentle bump that this one is good to go now 🙂

**Review by jszwedko — APPROVED** [review]
Nifty! Thanks again for this @britton-from-notion . It handles resizing much better now and, in general, looks more polished.

**Comment by jszwedko** [comment]
Force merging since integration test failure is unrelated (seems to be an issue on `master` that I'll look into).

**Pull request merged**